client/src/components/Account/UploadPhoto/UploadPhoto.tsx

```diff
@@ -1,12 +1,88 @@
-import { Box, Typography } from '@material-ui/core';
+import { Box, Typography, Avatar, Grid, Button, FormControl } from '@material-ui/core';
+import DeleteIcon from '@material-ui/icons/Delete';
+import SaveIcon from '@material-ui/icons/Save';
+
+import React, { ChangeEventHandler, useRef, useState } from 'react';
```

Comment:
```suggestion
import React, { useRef, useState } from 'react';
```
You can remove this unused import